e2e/test/scenarios/question/column-compare.cy.spec.ts

```diff
@@ -433,7 +434,8 @@ describeWithSnowplow("scenarios > question > column compare TODO", () => {
       verifyNoColumnCompareShortcut();
       cy.realPress("Escape");
 
-      openVisualization();
+      cy.button("Show Visualization").click();
+      queryBuilderMain().findByText("42").should("be.visible");
```

Comment:
new: wait for the visualization to be shown

previously things happened too fast